mod-orders/examples/bindPiecesCollection.sample

```diff
@@ -0,0 +1,26 @@
+{
+  "toBeBound": [
```

Comment:
Good that we follow an existing approach of payload like to beReceieved etc
I also think that bound, totalRecords are redundant. and in previous payloads as well.
If we remove these 2 fields we will make this payload inconsistent with other similar, so let's keep